python/ray/serve/_private/http_proxy.py

```diff
@@ -352,10 +352,16 @@ def update_draining(self, draining: bool):
         """
 
         if draining and (not self._is_draining()):
-            logger.info(f"Start to drain the proxy actor on node {self._node_id}.")
+            logger.info(
+                f"Start to drain the proxy actor on node {self._node_id}.",
+                extra={"log_to_stderr": False},
```

Comment:
@edoakes Could you confirm whether these logs should stop being printed to the terminal?

Reply:
yea looks good